tests/foreman/cli/test_errata.py

```diff
@@ -1091,22 +1092,28 @@ def test_positive_list_filter_by_product_id_and_org_label(self):
             set([])
         )
 
-    @stubbed()
-    def test_positive_list_filter_by_product_name(self):
```

Comment:
@oshtaier , because there should not be a positive list by product name only and the list with product name should lead to exception, and the test has been renamed to negative